<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherTestSettings</class>
 <widget class="QDialog" name="OtherTestSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>459</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Параметры тестирования</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Время удержания&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::Section::SecondSection</enum>
       </property>
       <property name="displayFormat">
        <string>mm:ss</string>
       </property>
       <property name="currentSectionIndex">
        <number>1</number>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>10</second>
        </time>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Значения для тестирования</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listWidget_value">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::CurrentChanged|QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_add_value">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete_value">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_change_value">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Шаги тестирования</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidget_step">
        <item>
         <property name="text">
          <string>3.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_add_step">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete_step">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_change_step">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Начать тест</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>OtherTestSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>OtherTestSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
